test-java-lib-lombok/src/main/java/com/uber/lombok/LombokBuilderInit.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.lombok;
+
+import javax.annotation.Nullable;
+import lombok.Builder;
+
+@Builder
+public class LombokBuilderInit {
+  private String field;
+  @Builder.Default private String fieldWithDefault = "Default";
+  @Nullable private String nullableField;
+}
```

Comment:
For reference, this becomes:

```
package com.uber.lombok;

import javax.annotation.Nullable;
import lombok.Builder;

@Builder()
public class LombokBuilderInit {
    private String field;
    @lombok.Builder.Default()
    private String fieldWithDefault;
    @Nullable()
    private String nullableField;
    
    @java.lang.SuppressWarnings(value = "all")
    private static String $default$fieldWithDefault() {
        return "Default";
    }
    
    @java.lang.SuppressWarnings(value = "all")
    LombokBuilderInit(final String field, final String fieldWithDefault, @Nullable()
    final String nullableField) {
        super();
        this.field = field;
        this.fieldWithDefault = fieldWithDefault;
        this.nullableField = nullableField;
    }
    
    @java.lang.SuppressWarnings(value = "all")
    public static class LombokBuilderInitBuilder {
        @java.lang.SuppressWarnings(value = "all")
        private String field;
        @java.lang.SuppressWarnings(value = "all")
        private boolean fieldWithDefault$set;
        @java.lang.SuppressWarnings(value = "all")
        private String fieldWithDefault$value;
        @java.lang.SuppressWarnings(value = "all")
        private String nullableField;
        
        @java.lang.SuppressWarnings(value = "all")
        LombokBuilderInitBuilder() {
            super();
        }
        
        @java.lang.SuppressWarnings(value = "all")
        public LombokBuilderInit.LombokBuilderInitBuilder field(final String field) {
            this.field = field;
            return this;
        }
        
        @java.lang.SuppressWarnings(value = "all")
        public LombokBuilderInit.LombokBuilderInitBuilder fieldWithDefault(final String fieldWithDefault) {
            this.fieldWithDefault$value = fieldWithDefault;
            fieldWithDefault$set = true;
            return this;
        }
        
        @java.lang.SuppressWarnings(value = "all")
        public LombokBuilderInit.LombokBuilderInitBuilder nullableField(@Nullable()
        final String nullableField) {
            this.nullableField = nullableField;
            return this;
        }
        
        @java.lang.SuppressWarnings(value = "all")
        public LombokBuilderInit build() {
            String fieldWithDefault$value = this.fieldWithDefault$value;
            if (!this.fieldWithDefault$set) fieldWithDefault$value = LombokBuilderInit.$default$fieldWithDefault();
            return new LombokBuilderInit(this.field, fieldWithDefault$value, this.nullableField);
        }
        
        @java.lang.Override()
        @java.lang.SuppressWarnings(value = "all")
        public java.lang.String toString() {
            return "LombokBuilderInit.LombokBuilderInitBuilder(field=" + this.field + ", fieldWithDefault$value=" + this.fieldWithDefault$value + ", nullableField=" + this.nullableField + ")";
        }
    }
    
    @java.lang.SuppressWarnings(value = "all")
    public static LombokBuilderInit.LombokBuilderInitBuilder builder() {
        return new LombokBuilderInit.LombokBuilderInitBuilder();
    }
}
```

before Error Prone gets to see it. As of 2.4.0, EP does not acknowledge `@SuppressWarnings("all")`.